sdk/src/azure/core/az_http_header_validation_private.h

```diff
@@ -41,7 +41,7 @@ static const uint8_t az_http_valid_token[256] = {
   0, // 16 ->  data link escape
   0, // 17 ->  device control 1
   0, // 18 ->  device control 2
-  0, // 19 ->  device control 3
+  0, // 19 ->  device control 3
```

Comment:
Lol, if anyone is wondering about what these ghostly changes are. We went from non-ascii-compliant non-breaking space character to the regular, 0x20 whitespace character :)